### PR TITLE
Fixes #6758: rudder-webapp upgrade doesn't commit updates to ncf hooks

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/6758

### CODE REVIEW

**Comment by amousset** [comment]
Missing change in debian packaging

**Comment by peckpeck** [comment]
Commit modified

File: rudder-webapp/debian/postinst [hunk 1]
```diff
@@ -112,21 +112,24 @@ case "$1" in
   find /var/rudder/configuration-repository/.git /var/rudder/configuration-repository/ncf /var/rudder/configuration-repository/techniques -type d -exec chmod g+s "{}" \;
 
   ## Add execution permission for ncf-api on pre/post-hooks
-  chmod -R 2750 /var/rudder/configuration-repository/ncf/ncf-hooks.d/
+  chmod -R 2750 %{ruddervardir}/configuration-repository/ncf/ncf-hooks.d
+  cd %{ruddervardir}/configuration-repository/ncf/
+  git add ncf-hooks.d
+  git commit --allow-empty --message "Add ne ncf hooks to repository"
```
Comment (amousset) [inline rudder-webapp/debian/postinst]: s/ne// ?

**Comment by peckpeck** [comment]
Commit modified

**Comment by peckpeck** [comment]
OK, merging this PR

**Pull request merged**